rotary/rotary.py

```diff
@@ -200,6 +238,21 @@ def parse_cli():
     parser_init.add_argument('-ns', '--no_short', action='store_true',
                              help="do not search for short read files")
     parser_init.set_defaults(init=True)
+
+    # =========================
+    # Declare Download Sub-command
+    download_help = """Downloads databases required by rotary to a specific directory """
+    parser_download = subparsers.add_parser('download', help=download_help)
+    parser_download.add_argument('-d', '--database_dir', metavar='PATH', required=True,
+                                 help='path the rotary database directory')
```

Comment:
Small grammar edit: should this be changed to `path to the rotary database directory`?

Reply:
Fixed